src/components/generic/form/ImageUpload.scss

```diff
@@ -74,9 +73,13 @@
         border-color: #777;
       }
 
+      &:focus {
+        color: #186df7;
+      }
+
       &:before {
         content: "+";
-        font-size: 2em;
+        font-size: 2.5rem;
```

Comment:
2.5 rem will look buggy on this repo. But it's meant to fit the h5pcom repo units (could enforce px but probably then should enforce px everywhere 🤔)